### PR TITLE
Fix#10 - Add name, surname and username editing

### PR DESCRIPTION
**Pull request by Arascog**
Fix#10->username

### CODE REVIEW

**Comment by Pryx** [comment]
First of all remove HTML comments, they are pretty unnecessary. Any future code comments should be in english. Will leave some comments in code.

**Comment by Pryx** [comment]
Also please rebase so it can be merged when this PR is finished.

File: classes/user.php [hunk 1]
```diff
@@ -392,6 +411,35 @@ public function render_user_settings()
...
+    global $mysqli, $message;
+    $id = $this->id;
+    $used = false;
+
+    $stmt = $mysqli->prepare("SELECT count(*) FROM users WHERE username LIKE ?");
+    $stmt->bind_param("s",$_POST["username"]);
+    $stmt->execute();
+    if ($stmt->num_rows > 0)
+    {
+        $used = true;
```
Comment (Pryx) [inline classes/user.php]: Please do an early return

File: classes/user.php [hunk 2]
```diff
@@ -392,6 +411,35 @@ public function render_user_settings()
...
+    }
+
+      if ($_SESSION['user'] != $id)
+      {
+        $message = _("Cannot change username of other users!");
+      }else{
+        if ($used)
```
Comment (Pryx) [inline classes/user.php]: This should be entirely gone after the change.

File: classes/user.php [hunk 3]
```diff
@@ -303,22 +303,41 @@ public function render_user_settings()
...
-      }else{
-        echo "<i class='fa fa-".($this->active?"check success":"times danger")."'></i>";
-      }?></div>
-    </div>
+    <form action="<?php echo WEB_URL;?>/admin/?do=user&amp;id=<?php echo $this->id; ?>" method="POST">
+      <div class="row">
+        <div class="col-md-2 col-md-offset-2"><strong><?php echo _("Username");?></strong></div>
+        <div class="col-md-6">
+          <div class="input-group">
+              <input type="text" class="form-control" name="username" required value=<?php echo $this->username." "; if ($this->id!=$_SESSION['user'] && $user->get_rank()<=1 && ($user->get_rank()<$this->rank))
```
Comment (Pryx) [inline classes/user.php]: Split this into multiple lines

**Comment by Pryx** [comment]
@Arascog any update? 🙂

**Comment by Arascog** [comment]
I don't have computer.. So I can't finish it. :/

**Pull request merged**